packages/prices-api/src/timestamp.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Converts a timestamp string or number to UTC unix timestamp in seconds
+ * @param timestamp - Timestamp as string (ISO format) or number (unix seconds)
+ * @returns UTC timestamp in seconds
+ * @example
+ * toUTC(1234567890) // Returns 1234567890
+ * toUTC("2024-01-01T00:00:00Z") // Returns 1704067200
+ * toUTC("2024-01-01T00:00:00.123Z") // Returns 1704067200 (ignores ms)
+ */
+export function toUTC(timestamp: string | number): number {
```

Comment:
why do we use timestamps instead of the `Date` object?

Reply:
Oh please do not use second timestamps that's confusing as JS usually expects milliseconds

Reply:
This function now returns a `Date` object, and all API model timestamps have been updated accordingly.